rules/follow-semver-when-publishing-npm-packages/rule.md

```diff
@@ -13,55 +13,52 @@ created: 2024-12-5T08:33:21.000Z
 guid: 03d090d3-d813-43d9-998e-209c895050b3
 ---
 
-When publishing an npm package, following **Semantic Versioning (SemVer)** is essential. It communicates changes clearly to your users and ensures smooth updates for their projects. This guide will tell you why adhering to SemVer is critical and how to do it effectively.  
+When publishing an npm package, following **Semantic Versioning (SemVer)** is essential. It communicates changes clearly to your users and ensures smooth updates for their projects.
 
 <!--endintro-->
 
 ## Why use SEMVER in NPM Publishing
 
 Semantic Versioning (SemVer) helps your users understand the impact of updates and manage their own dependencies more effectively. By adhering to SemVer, you make it clear whether an update introduces breaking changes, new features, or just bug fixes:
 
-- **MAJOR (Breaking Changes):** Signals to users that there are incompatible changes.
-- **MINOR (New Features):** Informs users about new features that won’t break existing functionality.
-- **PATCH (Bug Fixes):** Indicates that bug fixes or small improvements have been made without changing behavior.
-
-Without proper SemVer practices, users might face unexpected issues or be unable to safely upgrade their dependencies, which can result in losing the user.
+* **MAJOR (Breaking Changes):** Signals to users that there are incompatible changes.
+* **MINOR (New Features):** Informs users about new features that won’t break existing functionality.
+* **PATCH (Bug Fixes):** Indicates that bug fixes or small improvements have been made without changing behavior.
```

Comment:
Is there a reason to change "-" to "*"? to my understanding they are the same

Reply:
i think this is a bug/ some thing the editor do it by itself, i did not make the changes. but if it does not break anything, probably it is fine